### DISCUSSION TITLE
FastAPI connect SQL server

### DISCUSSION THREAD

**Discussion by sonpxp** [fastapi/fastapi#4919] [Questions]
## fastapi can connect SQLServer, if possible please give me an example, thanks a lot.

**Comment by nikstuckenbrock** [comment]
Hey @sonpxp,

it's described in the documentation und [SQL (Relational) databases](https://fastapi.tiangolo.com/tutorial/sql-databases/?h=sq#sql-relational-databases).
I hope I could help.
